isis/src/base/objs/ShapeModelFactory/ShapeModelFactory.truth

```diff
@@ -45,12 +45,12 @@ Unit test for Isis::ShapeModel
 
   Testing Isis cube file for dem that is not map projected
 **PROGRAMMER ERROR** Unable to create a shape model from given target and pvl.
-**I/O ERROR** Invalid shape model file [data/mgs/testData/ab102401.cub] in Kernels group.
+**I/O ERROR** Invalid shape model file [/usgs/cpkgs/isis3/isis_data/mgs/testData/ab102401.cub] in Kernels group.
```

Comment:
These line contain full paths "/usgs/cpkgs". There should be a regex in the unit test that is removing these so the tests will run with the data being in this and only this path.